plugins/woocommerce/includes/data-stores/class-wc-product-variable-data-store-cpt.php

```diff
@@ -149,6 +159,7 @@ public function read_children( &$product, $force_read = false ) {
 			}
 			$children['all']     = get_posts( apply_filters( 'woocommerce_variable_children_args', $all_args, $product, false ) );
 			$children['visible'] = get_posts( apply_filters( 'woocommerce_variable_children_args', $visible_only_args, $product, true ) );
+			$children['version'] = WC_Cache_Helper::get_transient_version( 'product' );
```

Comment:
Would this be equivalent to:

```suggestion
			$children['version'] = $transient_version;
```

Reply:
thanks! done in https://github.com/woocommerce/woocommerce/pull/53603/commits/7d9b4bb3f13faa2840167d32308fbb06e74854ef